draft-ietf-core-senml.md

```diff
@@ -983,6 +993,12 @@ from 19th to the last:
 
   coap://example.com/temp#rec=3-5,10,19-*
 
+## Fragment Identification for the XML Format
+
+In addition to the SenML Fragment Identifiers described above, with 
+the XML SenML formats also the syntax defined in {{XPointerFramework}}
```

Comment:
Does "XML SenML formats" include EXI?

Reply:
Good question. I suppose same fragment IDs work for EXI? If yes, could make it explicit here.

Reply:
So see my PR to this PR...

Reply:
#117, that is.